installation/template/js/remove.js

```diff
@@ -22,7 +22,33 @@ if (document.getElementById('skipLanguages')) {
 		document.getElementById('installLanguages').classList.remove('active');
 	})
 }
-
+if (document.getElementById('removeInstallationFolder')) {
```

Comment:
Ah, I forgot something: Empty line before the "if" which was deleted should be added back. Code style.

```suggestion

if (document.getElementById('removeInstallationFolder')) {
```